app/queries/user_analytics_query.rb

```diff
@@ -1,98 +0,0 @@
-class UserAnalyticsQuery
```

Comment:
If I understand correctly, this stuff has moved to the presenter, which leverages simple aggregations on the `follow_ups` relation?

Reply:
Yeah, the Query abstraction was useful before because we were doing more complex queries. Not they are a lot simpler.